app-setup.sh

```diff
@@ -25,7 +25,7 @@ chmod +x app/.profile
     chmod +x app/cloud-service-broker
 
 # Add the brokerpak(s)
-(cd app && curl -f -LO https://github.com/GSA/eks-brokerpak/releases/download/v0.6.0/eks-services-pack-0.6.0.brokerpak)
+(cd app && curl -f -LO https://github.com/GSA/eks-brokerpak/releases/download/v0.7.0/eks-services-pack-0.7.0.brokerpak)
```

Comment:
🤷  given that the version lives in two places in the URL, I'd consider moving this up to a variable at the top of the file.

Reply:
There's not really much value in having a separate PR every time I make an upstream release to eks-brokerpak... They're essentially black boxes from the SSB's perspective, and you're not going to catch any changes in behavior through review in this repo.

I kind of want to remove the explicit version number and just use the latest eks-brokerpak release, and figure out how to trigger the GitHub Action every time there's a new release in eks-brokerpak.

(This is analogous to wanting new versions of inventory-app, catalog-app, etc. ship in continuously via datagov-deploy without manual intervention to make that happen.)